voluncheer/profiles/views/volunteers.py

```diff
@@ -23,7 +23,7 @@ def get_context_data(self, **kwargs):
     def form_valid(self, form):
         """Saves the new user and logs them in."""
         user = form.save()
-        user.is_active = False
+        user.is_active = True
```

Comment:
Wouldn't this defeat the purpose of email activation?

Reply:
Ahh, thanks for pointing out. I forget to change back